lib/werewolf/slackbot.rb

```diff
@@ -3,6 +3,24 @@
 module Werewolf
   class SlackBot < SlackRubyBot::Server
     attr_writer :channel
+    cattr_reader :role_icons
+
+    @@role_icons = {
```

Comment:
this is, in fact, a constant.  right?  you wouldn't want someone to change this at runtime?